tests/Integration/Monitoring/Test_Plotter.py

```diff
@@ -0,0 +1,404 @@
+from functools import reduce
+from six.moves import map
+
+
+def compare(file1Path, file2Path):
+  '''
+    Function used to compare two plots
+
+    returns 0.0 if both are identical
+  '''
+  # Crops image to remove the "Generated on xxxx UTC" string
+  image1 = Image.open(file1Path).crop((0, 0, 800, 570))
+  image2 = Image.open(file2Path).crop((0, 0, 800, 570))
+  h1 = image1.histogram()
+  h2 = image2.histogram()
+  rms = math.sqrt(reduce(operator.add,
+                         list(map(lambda a, b: (a - b) ** 2, h1, h2))) / len(h1))
+  return rms
+
+
+class PlotterTestCase(unittest.TestCase):
+
+  def setUp(self):
+    pass
+
+  def tearDown(self):
+    pass
+
+
+class ComponentMonitoringPlotterUnitTest(PlotterTestCase):
+  """ First you will need to generate plots on the DIRAC WebApp and then add it to the
+      DIRAC/tests/Integration/Monitoring/png/ folder.
+      Then change the data dictionary here according to the WebApp plot and then run this regression test.
+  """
+
+  def test_plotRunningThreads(self):
+    """ test for the method "_plotRunningThreads"
+    """
+
+    plotName = "ComponentMonitoringPlotter_plotRunningThreads"
+    reportRequest = {'grouping': 'component',
+                     'groupingFields': ('%s', ['component']),
+                     'startTime': 1562796000,
+                     'endTime': 1562882400,
+                     'condDict': {'component': 'Monitoring_Monitoring'}
+                     }
+
+    plotInfo = {'data': {'Monitoring_Monitoring': {1562853600: 9.0,
+                                                   1562857200: 0,
+                                                   1562854500: 0,
+                                                   1562855400: 0,
+                                                   1562856300: 0,
+                                                   1562858100: 0,
+                                                   1562849100: 9.75,
+                                                   1562860800: 0,
+                                                   1562850000: 10.0,
+                                                   1562848200: 9.142857142857142,
+                                                   1562859900: 0,
+                                                   1562850900: 9.428571428571429,
+                                                   1562851800: 9.0,
+                                                   1562847300: 8.5,
+                                                   1562852700: 9.0}
+                         },
+                'unit': 'threads',
+                'granularity': 900}
+
+    obj = ComponentMonitoringPlotter(None, None)
+    res = obj._plotRunningThreads(reportRequest, plotInfo, plotName)
+    self.assertEqual(res['OK'], True)
+    self.assertEqual(res['Value'], {'plot': True, 'thumbnail': False})
+
+    res = compare('%s.png' % plotName, '/opt/dirac/DIRAC/tests/Integration/Monitoring/png/%s.png' % plotName)
```

Comment:
please do not use /opt/dirac/ I already said you can install DIRAC to different directory ...

Reply:
Yes I changed that but there was a travis issue which has to be merged so will push the changes.